Bottom: 93d662df816b39cad3d733f7b6601f6235a5cf16
Top:    88dc8dec6590a977ab1737e8fff672705f27d651
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-09-01 17:57:14 +0800

Refresh of vhost-fix-busy-polling-in-the

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index c69da67..afd0480 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -426,7 +426,6 @@ static bool vhost_net_tx_avail(struct vhost_net *net,
 		cpu_relax();
 	preempt_enable();
 
-	printk("vq avail %d\n", vhost_vq_avail_empty(vq->dev, vq));
 	return !vhost_vq_avail_empty(vq->dev, vq);
 }
 
@@ -655,7 +654,7 @@ static int vhost_net_rx_peek_head_len(struct vhost_net *net, struct sock *sk)
 
 		preempt_enable();
 
-		if (vhost_enable_notify(&net->dev, vq))
+		if (!vhost_vq_avail_empty(&net->dev, vq))
 			vhost_poll_queue(&vq->poll);
 		mutex_unlock(&vq->mutex);
